src/NBomber/InternalExtensions.fs

```diff
@@ -73,13 +74,38 @@ module internal Extensions =
             if Array.isEmpty array then defaultValue
             else array |> Array.averageBy f
 
+        // shuffle an array (in-place)
+        let shuffle a =
+            let swap (a: _[]) x y =
+                let tmp = a.[x]
+                a.[x] <- a.[y]
+                a.[y] <- tmp
+            let rand = Random()
+            Array.iteri (fun i _ -> swap a i (rand.Next(i, Array.length a))) a
+            a
+
     module Map =
 
         let inline fromDictionary (dictionary) =
             dictionary
             |> Seq.map (|KeyValue|)
             |> Map.ofSeq
 
+    type Dict<'k, 'v> = System.Collections.Generic.IDictionary<'k,'v>
+
+     module Dict =
+        let isEmpty(dictionary : Dict<'k,'a>)  =
```

Comment:
inline ?

Reply:
please explain, eliminate alias?

Reply:
let inline like you did for Map
let inline fromDictionary (dictionary) =

Reply:
done

Reply:
nop, `inline` doesn't compile, removed it again